package.json

```diff
@@ -68,7 +68,9 @@
     "whatwg-fetch": "1.0.0"
   },
   "devDependencies": {
+    "babel-preset-jest": "^14.0.0",
     "bundle-deps": "1.0.0",
+    "jest": "^14.0.0",
```

Comment:
Should these two be regular dependencies of `react-scripts`? Otherwise I’m not sure how they would get installed in user’s project.